network-api/networkapi/templates/donate/pages/help_page.html

```diff
@@ -24,7 +24,7 @@ <h1 class="tw-h1-heading"> {{ page.title }} </h1>
             <div class="tw-border-t tw-border-b tw-py-10 tw-mt-28 tw-mb-10 tw-border-gray-20">
                 <p id="contact-us" class="tw-h3-heading">{% trans "Contact Us" %}</p>
                 {% if show_formassembly_thank_you %}
-                    <div class="tw-py-22">
+                    <div id="thank-you" class="tw-py-22">
```

Comment:
Added this ID so I can easily locate this div in [001-help-form-spec.js](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/e1fb131ca01dd43b200c7c02c1b87a929320a3c0/tests/integration/donate/help/001-help-form.spec.js#L156)